packages/fuels/tests/providers.rs

```diff
@@ -517,8 +517,9 @@ async fn test_gas_errors() -> Result<()> {
         .await
         .expect_err("should error");
 
-    let expected = "Provider error: Response errors; not enough coins to fit the target";
-    assert!(response.to_string().starts_with(expected));
+    // let expected = "Provider error: Response errors; not enough coins to fit the target";
+    dbg!(&response.to_string());
```

Comment:
Is this a leftover `dbg!` statement? Also, the `assert!` is commented out -- is it by design?

Reply:
No I just forgot to uncomment that.